doc/Yang Model Guidelines.md

```diff
@@ -140,27 +143,18 @@ container VLAN_MEMBER {
 }
 ```
 
-### 5.  It is best to categorized Yang Modules based on a networking components. For Example, it is good to have separate modules for VLAN,  ACL, PORT and IP-ADDRESSES etc.
+### 5.  It is best to categorized YANG Modules based on a networking components. For Example, it is good to have separate modules for VLAN,  ACL, PORT and IP-ADDRESSES etc.
```

Comment:
Please change to ... "It is best to categorize YANG Modules based on a networking components. For example ......."